From 5498e8cdaf7cead5be748a4a41df520dbef3822f Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Fri, 2 Jun 2017 17:09:43 +0200
Subject: [PATCH 2/7] RunGrepTest: compare without paying attention to line
 endings

The output of pcre2grep has CR/LF line endings on Windows, but PCRE2's
tests expect LF-only.

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 RunGrepTest | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/RunGrepTest b/RunGrepTest
index 396884cc..f5641292 100755
--- a/RunGrepTest
+++ b/RunGrepTest
@@ -58,10 +58,11 @@ fi
 # that lacks a -u option. Try to deal with this; better do the test for the -b
 # option as well.
 
-cf="diff"
-diff -b  /dev/null /dev/null 2>/dev/null && cf="diff -b"
-diff -u  /dev/null /dev/null 2>/dev/null && cf="diff -u"
-diff -ub /dev/null /dev/null 2>/dev/null && cf="diff -ub"
+#cf="diff"
+#diff -b  /dev/null /dev/null 2>/dev/null && cf="diff -b"
+#diff -u  /dev/null /dev/null 2>/dev/null && cf="diff -u"
+#diff -ub /dev/null /dev/null 2>/dev/null && cf="diff -ub"
+cf="git -c core.autocrlf=false diff --no-index --ignore-space-at-eol"
 
 # Add a -a (always treat as text) if available. This was added in an attempt
 # to get more detail from an Alpine Linux test failure on GitHub.
-- 
2.48.1.windows.1

